addons/event/models/event_event.py

```diff
@@ -237,7 +237,7 @@ def _compute_seats(self):
             self._cr.execute(query, (tuple(self.ids),))
             res = self._cr.fetchall()
             for event_id, state, num in res:
-                results[event_id][state_field[state]] += num
+                results[event_id][state_field[state]] = num
```

Comment:
Small note: this is not usefull for the fix, but makes the code easier to understand.